content/docs/user-guide/basic-concepts/parameter.md

```diff
@@ -2,8 +2,7 @@
 name: 'Parameter Dependency'
 match: [parameter, parameters, param, params, hyperparameter, hyperparameters]
 tooltip: >-
-  Pipeline stages (defined in `dvc.yaml`) can depend on specific values inside
-  an arbitrary YAML, JSON, TOML, or Python file (`params.yaml` by default).
-  Stages are invalid (considered outdated) when any of their parameter values
-  change. See [`dvc params`](/doc/command-reference/params).
+  Pipeline stages (defined in `dvc.yaml`) can depend on specific values inside a
```

Comment:
Again, here is an example when it's a bit too low level DVC-specific and not meaningful for users who are not deeply familiar with DVC already. We should be using more of a domain language, or have some connection to actual things besides DVC (can depend on hyperparameters, training parameters, any custom processing parameters ...)

Reply:
Hmmm. I can't use the word "parameters" to define "parameter". How about `Simple values that your code can depend on`? I think it's good to be specific here since this is a definition, not some page that will come up in search results. Use cases and motivation are provided in all the actual docs.

Reply:
I would make (including model parameters, etc) - it's fine in this case to use "something parameters" to kinda clarify what do we means by parameters

Reply:
Also now in #3899